src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1049,6 +1049,9 @@ public List<SyntaxNode> GetMethodLevelMembers(SyntaxNode root)
         public SyntaxList<SyntaxNode> GetMembersOfTypeDeclaration(SyntaxNode typeDeclaration)
             => ((TypeDeclarationSyntax)typeDeclaration).Members;
 
+        public SyntaxList<SyntaxNode> GetMembersOfNamespaceDeclaration(SyntaxNode namespaceDeclaration)
```

Comment:
so this can include other namespaces I guess? do we have test case where namespace has mix of type and namespace as member?

Reply:
Yes, this can include other namespaces. No tests with mix, that's a good case to add.